templates/includes/header.html

```diff
@@ -13,7 +13,9 @@
         <div class="section-wrap">
             <input id="{{ link.section }}-submenu-checkbox" type="checkbox" class="submenu-checkbox" aria-hidden="true"
                 {{ util::current_checked(if_path_starts_with="/" ~ link.section) }}>
-            <label for="{{ link.section }}-submenu-checkbox" class="submenu-title">{{ link.title }} <div class="arrow">
+            <label for="{{ link.section }}-submenu-checkbox" class="submenu-title">
+            <a href="/{{ link.section }}">
+                {{ link.title }}</a> <div class="arrow">
```

Comment:
this makes the section indexes navigable, which has an impact on existing sections:
- clicking on "Foundation" forwards you to /foundation/about/
- clicking on "Ecosystem" lands you at the previously not navigable /ecosystem index page